src/components/ui/StepView/StepView.tsx

```diff
@@ -0,0 +1,56 @@
+import styles from "./StepView.module.scss";
+import cs from "classnames";
+import React from "react";
+import NextArrow from "../../../assets/vectors/next-arrow.svg"
+
+interface StepViewProps {
+    steps: string[]
+    activeStepIndex: number
+}
+
+interface ClickableStepViewProps {
+    steps: string[]
+    activeStepIndex: number
+    onStepClicked: (index: number) => void
+}
+
+export const ClickableStepView: React.FC<ClickableStepViewProps> = ({steps, activeStepIndex, onStepClicked}) => {
+    return (
+        <div className={styles.clickableSteps}>
+            {steps.map((step, i) => {
+                return <div key={i.toString()} onClick={() => onStepClicked(i)} className={styles.clickableStepContainer}>
+                    {singleStep(i + 1, step, i === activeStepIndex)}
+                </div>
+            })}
+        </div>
+    )
+}
```

Comment:
it doesn't seems to be used anywhere

Reply:
i think we can avoid making clickable copy-paste of original by moving creating steps to view itself
ex.
```jsx
<StepsList>
   <Step label={firstStep} active={step == 0}>
   <Step label={secoundStep} active={step == 1}>
   <Step label={otherStep} active={step == 2}>
<StepsList/>
```
then making onClick prop or wapper would be easy as hell

Reply:
I'll give it a try